src/components/Swap/Chart.vue

```diff
@@ -109,12 +124,23 @@ import { Components } from '@/consts';
 import { debouncedInputHandler, getTextWidth, calcPriceChange, formatDecimalPlaces } from '@/utils';
 
 import type { AccountAsset } from '@sora-substrate/util/build/assets/types';
+import type {
+  PageInfo,
+  AssetSnapshotEntity,
+  FiatPriceObject,
+} from '@soramitsu/soraneo-wallet-web/lib/services/subquery/types';
+
+// open, close, low, high
+type OCLH = [number, number, number, number];
 
-type ChartDataItem = {
+type SnapshotItem = {
   timestamp: number;
-  price: number[];
+  price: OCLH;
 };
 
+// timestamp, open, close, high, low
+type ChartDataItem = [number, ...OCLH];
```

Comment:
the same issue, please use JS Doc format to have highlighted description in the code editor